crates/server/src/admin/handlers/identity/generate_context_identity.rs

```diff
@@ -1,17 +1,46 @@
 use std::sync::Arc;
 
+use axum::http::StatusCode;
 use axum::response::IntoResponse;
 use axum::Extension;
+use calimero_primitives::context::ContextId;
 use calimero_server_primitives::admin::GenerateContextIdentityResponse;
+use calimero_store::types::ContextIdentity as ContextIdentityValue;
+use tracing::error;
 
-use crate::admin::service::ApiResponse;
+use crate::admin::service::{ApiError, ApiResponse};
 use crate::AdminState;
 
+const PLACEHOLDER_CONTEXT_ID_BYTES: [u8; 32] = [0; 32];
+
 pub async fn handler(Extension(state): Extension<Arc<AdminState>>) -> impl IntoResponse {
     let private_key = state.ctx_manager.new_private_key();
+    let public_key = private_key.public_key();
+
+    let store_result = (|| {
+        let placeholder_context_id = ContextId::from(PLACEHOLDER_CONTEXT_ID_BYTES);
+        let value = ContextIdentityValue {
+            private_key: Some(*private_key),
+            sender_key: None,
+        };
+        state
+            .ctx_manager
+            .store_identity_value(placeholder_context_id, public_key, value)?;
+        Ok::<(), eyre::Report>(())
+    })();
```

Comment:
this should be the responsibility of some function inside context manager (so it can be used by merod's interactive CLI too)